bokehjs/src/lib/models/widgets/paragraph.ts

```diff
@@ -8,7 +8,15 @@ export class ParagraphView extends MarkupView {
   override render(): void {
     super.render()
     // This overrides default user-agent styling and helps layout work
-    const content = paragraph({style: {margin: 0}}, this.model.text)
+    let content: HTMLParagraphElement
+
+    if (this.has_math_disabled())
+      content = paragraph({style: {margin: 0}}, this.model.text)
+    else {
+      content = paragraph({style: {margin: 0}})
+      content.innerHTML = this.process_tex()
+    }
```

Comment:
Try keeping `const` whenever possible (and avoid duplication at the same time), e.g.:
```ts
    const content = paragraph({style: {margin: 0}})

    if (this.has_math_disabled())
      content.textContent = this.model.text
    else
      content.innerHTML = this.process_tex()
```